app/src/main/java/com/group80/uoftinder/login_use_case/LoginUser.java

```diff
@@ -0,0 +1,41 @@
+package com.group80.uoftinder.login_use_case;
+
+import android.widget.EditText;
+
+import com.google.firebase.auth.FirebaseAuth;
+
+public class LoginUser {
```

Comment:
This class is accually not used anywhere. I think it collides with the `entities.User`. Should we delete this file?

Reply:
File was deleted